nautobot/dcim/tests/test_api.py

```diff
@@ -69,6 +69,7 @@
     VirtualDeviceContext,
 )
 from nautobot.extras.models import ConfigContextSchema, Role, SecretsGroup, Status
+from nautobot.extras.models.models import ExternalIntegration
```

Comment:
Could probably just do this here.
```suggestion
from nautobot.extras.models import ExternalIntegration
```